tests/unit/generator_test.py

```diff
@@ -30,9 +31,8 @@ def generate(model: Model) -> Generator:
     return generator
 
 
-@pytest.mark.skipif(not __debug__, reason="depends on assertion")
 def test_invalid_prefix() -> None:
-    with pytest.raises(AssertionError, match=r"empty part in identifier"):
+    with pytest.raises(RecordFluxError, match=r"empty part in identifier"):
```

Comment:
Please match error message completely using `^` and `$`.

Reply:
Added in 6e1f072